### ISSUE TITLE
Error in Colab

### ISSUE THREAD

**Issue by icarosun** [awslabs/autogluon#1948]
**Describe the bug**
I ran this command in Colab: 

`!pip install autogluon`

Then, I import TextPredictor:

`from autogluon.text import TextPredictor`

unfortunately when I ran this command it printed this wrong:

![image](https://user-images.githubusercontent.com/46682140/178778840-7a11447e-665f-4a32-bbdb-c62b2b082d79.png)

**Expected behavior**
That program run with no errors and I can fit the model with TextPredictor.

**To Reproduce**
https://colab.research.google.com/drive/1QYXD0QzC5p9Dl9v6Aw_ptMXM1I_aagwI?usp=sharing

**Info**
No wrongs with autogluon.tabular

**Comment by sxjscience** [comment]
```bash
!pip freeze
```

Output

```
torch==1.11.0
torchaudio @ https://download.pytorch.org/whl/cu113/torchaudio-0.12.0%2Bcu113-cp37-cp37m-linux_x86_64.whl
torchmetrics==0.7.3
torchsummary==1.5.1
torchtext==0.13.0
torchvision==0.12.0
```

**Comment by zhiqiangdon** [comment]
@icarosun This issue has been fixed in the 0.5.1 release.

**Comment by sxjscience** [comment]
Feel free to reopen if you still find it's an issue.
